server/src/main/resources/file.conf

```diff
@@ -82,11 +82,7 @@ store {
 }
 lock {
     ## the data row lock store mode: local_db、memory or db
-    mode = "memory"
-
-    memory{
-         ## store lock in memory of server
-    }
+    mode = ""
```

Comment:
should we remove lock.mode from conf

Reply:
> should we remove lock.mode from conf

I'll use “lock.mode”  in some other cases.